libs/model/src/config.ts

```diff
@@ -593,7 +593,7 @@ export const config = configure(
     }),
     MCP: z.object({
       MCP_DEMO_CLIENT_SERVER_URL: z.string().optional(),
-      MCP_DEMO_CLIENT_KEY: z.string().optional(),
+      MCP_KEY_BYPASS: z.string().optional(),
```

Comment:
Just for safety should probably add refine step to make sure this doesn't get set in production.